gammapy/modeling/models/cube.py

```diff
@@ -332,7 +336,9 @@ def evaluate_geom(self, geom, gti=None):
             value = value * self.spatial_model.evaluate_geom(geom)
 
         if self.temporal_model:
-            integral = self.temporal_model.integral(gti.time_start, gti.time_stop)
+            integral = self.temporal_model.integral(
```

Comment:
here you want to have the integral over time at given true energies. Not the integral over time and true energy, right?